examples/cryptocurrency-advanced/backend/src/api.rs

```diff
@@ -37,7 +37,8 @@ pub struct WalletProof {
     /// Proof of the whole database table.
     pub to_table: MapProof<IndexCoordinates, Hash>,
     /// Proof of the specific wallet in this table.
-    pub to_wallet: MapProof<PublicKey, Wallet>,
+    //TODO: revert change to Raw
```

Comment:
Will this task be performed separately? Also, not sure why it is necessary. (Do we consider `RawProofMapIndex` too difficult for a tutorial / demo?)

Reply:
No, it's debug artefact, forgot to remove it.